lib/paper_trail/background/config.rb

```diff
@@ -0,0 +1,23 @@
+module PaperTrail
+  module Background
+    class Configuration
+      attr_reader :opt_in
+
+      def opt_in=(value)
+        @opt_in = value
+      end
+    end
+
+    class Config
+      class << self
+        def configuration
+          @configuration ||= Configuration.new
+        end
+
+        def configure(&block)
+          yield(configuration)
+        end
+      end
+    end
```

Comment:
Bleh, I hate this style of configuration.

Reply:
Can you elaborate and offer an alternative? :-)

Reply:
No.

Reply:
@seejohnplay Oh you work with the contributor, so it's only slightly weird of you to comment randomly on a merged PR you aren't a part of.